python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -517,3 +509,100 @@ def to_arrow(self):
                 arrow_polygons,
             ],
         )
+
+    def _align_to_index(
+        self: T,
+        index: ColumnLike,
+        how: str = "outer",
+        sort: bool = True,
+        allow_non_unique: bool = False,
+    ) -> T:
+        # The values in the newly aligned columns will not change,
+        # only their positions in the union offsets.
```

Comment:
Use `"""` for function wide comments
```suggestion
        """
        The values in the newly aligned columns will not change,
        only positions in the union offsets and type codes.
        """
```